spec/classes/init_spec.rb

```diff
@@ -1280,4 +1270,25 @@
       end
     end
   end
+
+  describe 'with pid_file parameter' do
+    context 'with pid_file set to absent' do
+      let(:params) { { :pid_file => 'absent' } }
+
+      it {
+        should_not contain_file('nrpe_config').with_content(/^pid_file=.*$/)
```

Comment:
better use
<pre>
should contain_file('nrpe_config').without_content(/^pid_file=.*$/)
</pre>

like here: https://github.com/Phil-Friderici/puppet-module-postfix/blob/ac0fa2fc802101dbe6e0c92056f34247090246f5/spec/classes/init_spec.rb#L108